projects/v2/src/app/app-config.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable }  from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { ReplaySubject } from 'rxjs';
+import { SheetService } from './services/sheet.service';
+import { SheetDetails } from './models/sheet.model';
+ 
+@Injectable()
+export class ConfigService {
+
+    config = new ReplaySubject(2);
+    sheetConfiguration = new ReplaySubject<SheetDetails[]>(2);
```

Comment:
sheetConfiguration => sheetConfiguration$

Reply:
config$ = this.http.get<SheetDetails[]>('assets/sheet-config.json').pipe(shareReplay(1));

Reply:
you might need a take(1) in the pipe call, but I'm pretty sure you don't need it.